themes/tdr/login/template.ftl

```diff
@@ -56,6 +59,16 @@
 
             <#-- Start TDR Content -->
             <div class="govuk-width-container">
+
+                <div class="govuk-phase-banner">
+                    <p class="govuk-phase-banner__content">
+                        <strong class="govuk-tag govuk-phase-banner__content__tag">${msg("betaBanner")}</strong>
+                        <span class="govuk-phase-banner__text">${msg("betaBannerInfo")}
+                <a class="govuk-link" target="_blank" rel="noreferrer noopener" href="${properties.tdrHomeUrl}/contact">${msg("betaBannerLink")}</a>
```

Comment:
This page doesn't exist yet, so the link will be broken. We shouldn't merge PRs with broken features because they might confuse users, so there are a few options:

1. Take the Beta banner out of this PR, and add it back once the contact page exists
2. Make the Beta banner look like the current main site (i.e. a message without a link), and add the link later
3. Hold off on merging this whole PR until the contact page exists

I'd rather do option 1 or 2 so that we can merge the other changes, but we can do option 3 if you know the contact page will be added very soon. What do you think?

Reply:
Thank you, Suzanne. I was going to merge the PR in the frontend project with the contact page (https://github.com/nationalarchives/tdr-transfer-frontend/pull/500) first, then merge this one. Would that be okay? I see why options 1 and 2 are a lot safer though! I can still row back and go with option 1. Happy with either, can you let me know what you prefer please :)

Reply:
OK, if the contact page is going to be merged soon then that seems OK.

I'm trying to avoid the situation where PRs sit open for a week because they're not ready to merge. As a reviewer, it's hard to figure out what I can review and what's blocked. (Look at how long the daily PR list has got!) And it's harder to merge things if they've been open for a while because there are more likely to be merge conflicts. Small PRs that are easy to approve, merge and deploy are the ideal. 🙂

Reply:
You are absolutely right. Will make sure this doesn't happen again!

Reply:
Just merged the front end pull request which means the contact page exists now and the link should be working.

Reply:
🎉